### PR TITLE
bsdmainutils 9.0.12+nmu1

### PR DESCRIPTION
**Pull request by rwhogg**
Signed-off-by: Bob W. Hogg <rwhogg@linux.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Linuxbrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/bsdmainutils.rb [hunk 1]
```diff
@@ -1,22 +1,25 @@
 class Bsdmainutils < Formula
   desc "Collection of utilities from FreeBSD"
   homepage "https://packages.debian.org/sid/bsdmainutils"
-  url "http://ftp.debian.org/debian/pool/main/b/bsdmainutils/bsdmainutils_9.0.12.tar.gz"
-  mirror "https://mirror.csclub.uwaterloo.ca/debian/pool/main/b/bsdmainutils/bsdmainutils_9.0.12.tar.gz"
-  sha256 "9b6989932eabb43fa0137f4e5589e9b1ec70ea0249276a31dd311b3664bfc97f"
+  url "http://ftp.debian.org/debian/pool/main/b/bsdmainutils/bsdmainutils_9.0.12+nmu1.tar.gz"
+  mirror "https://mirror.csclub.uwaterloo.ca/debian/pool/main/b/bsdmainutils/bsdmainutils_9.0.12+nmu1.tar.gz"
+  version "9.0.12+nmu1"
```
Comment (rwhogg) [inline Formula/bsdmainutils.rb]: This is necessary; otherwise it thinks the version number is 1 (probably from the "+nmu1" part).

**Comment by maxim-belkin** [comment]
Circle:
```
==> brew bottle --verbose --json bsdmainutils --keep-old
==> Bottling bsdmainutils-9.0.12+nmu1.x86_64_linux.bottle.tar.gz...
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/bsd-from
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/bsd-write
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/calendar
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/col
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/colcrt
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/colrm
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/column
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/hexdump
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/look
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/ncal
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/printerbanner
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath @@HOMEBREW_PREFIX@@/Cellar/bsdmainutils/9.0.12+nmu1/lib:@@HOMEBREW_PREFIX@@/lib:@@HOMEBREW_PREFIX@@/opt/ncurses/lib:@@HOMEBREW_PREFIX@@/opt/libbsd/lib --set-interpreter /lib64/ld-linux-x86-64.so.2 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/ul
/usr/bin/touch -h -t 201704121122.14 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/cal
/usr/bin/touch -h -t 201704121122.14 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/hd
/usr/bin/touch -h -t 201704121122.14 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/share/man/man1/cal.1
/usr/bin/touch -h -t 201704121122.14 /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/share/man/man1/hd.1
tar cf /home/ubuntu/homebrew-core/bsdmainutils-9.0.12+nmu1.x86_64_linux.bottle.tar bsdmainutils/9.0.12+nmu1
gzip -f bsdmainutils-bottle.tar
Warning: String '/home/linuxbrew/.linuxbrew' still exists in these files:
/home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/calendar
 --> match '/home/linuxbrew/.linuxbrew/include/bsd' at offset 0xdc94
/home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/ncal
 --> match '/home/linuxbrew/.linuxbrew/include' at offset 0xb788
/home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/ul
 --> match '/home/linuxbrew/.linuxbrew/include' at offset 0x4c95

/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/bsd-from
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/bsd-write
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/calendar
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/col
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/colcrt
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/colrm
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/column
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/hexdump
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/look
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/ncal
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/printerbanner
/home/linuxbrew/.linuxbrew/opt/patchelf/bin/patchelf --set-rpath /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/libbsd/lib --set-interpreter /home/linuxbrew/.linuxbrew/lib/ld.so /home/linuxbrew/.linuxbrew/Cellar/bsdmainutils/9.0.12+nmu1/bin/ul
Warning: --keep-old is passed but there are changes in: cellar
./bsdmainutils-9.0.12+nmu1.x86_64_linux.bottle.tar.gz
  bottle do
    cellar :any_skip_relocation
    sha256 "76e633b1233ebd1b7dfea34bf4020b8986030082bbfda45f265d189458f1c61d" => :x86_64_linux
  end
==> brew bottle --merge --write --no-commit ./bsdmainutils-9.0.12+nmu1.x86_64_linux.bottle.json --keep-old
Error: --keep-old was passed but there are changes in:
sha256 => x86_64_linux
Error: Kernel.exit
==> bsdmainutils
==> FAILED
```

**Comment by rwhogg** [comment]
Yeah, looks like I forgot to tell Circle not to keep-old. I'll fix that.

**Comment by maxim-belkin** [comment]
why bother? it was successful, so just merge it... oh, well :)

**Comment by maxim-belkin** [comment]
green!

**Comment by rwhogg** [comment]
Closed in 019abfc23522c7365bde88cb4a51a6dd7ba2e72a

**Pull request closed without merging**